integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -131,6 +131,35 @@ journal:
 ## Worker ##
 
 # Tiered Storage
+# emptyDir example
+#  - level: 0
+#    mediumtype: MEM
+#    path: /dev/shm
+#    type: emptyDir
+#    quota: 1G
+#
+# hostPath example
+#  - level: 0
+#    mediumtype: MEM
+#    path: /dev/shm
+#    type: hostPath
+#    quota: 1G
+#
+# persistentVolumeClaim example
+#  - level: 1
+#    mediumtype: SSD
```

Comment:
is mediumtype `MEM` possible with pv? If not please explicitly state that in the doc page and why

Reply:
What this PVC's mediumtype should be really depends on what PV it binds to. So it can actually be `MEM` as `hostPath` and `emptyDir`. I put a level 1 `SSD` here just trying to illustrate it can have level > 0 like being a `SSD` or `HDD`.

Is there a particular scenario you are thinking about? Or how do you think this can be illustrated more clearly?